src/constants.js

```diff
@@ -207,7 +210,13 @@ export const SettingDefaultValues = {
   [SettingIds.EMOTE_AUTOCOMPLETE]: true,
   [SettingIds.BLACKLIST_KEYWORDS]: {},
   [SettingIds.HIGHLIGHT_KEYWORDS]: null,
-  [SettingIds.SIDEBAR]: [SidebarFlags.OFFLINE_FOLLOWED_CHANNELS | SidebarFlags.FEATURED_CHANNELS, 0],
+  [SettingIds.SIDEBAR]: [
+    SidebarFlags.OFFLINE_FOLLOWED_CHANNELS |
```

Comment:
could we build a migration step for old settings? some users who explicitly toggled these will have their settings wiped.

Reply:
A build migration step has been added into the file `src/settings.js` and in the function `upgradeFlags`.